documentation.md

```diff
@@ -133,11 +135,10 @@ the number of days by which the first payment will be deferred
 
 the number of months by which the first payment will be deferred
 
-## Customize CSS
+By default, the widget will display all your available payment plans.
 
-If you inspect the widgets, you can see two types of classes :
+## Customize CSS
 
-- one that looks gibberish
-- one starting with `alma-`
+If you want to customize the look of the widget, you can edit all classes that start with `alma-`
 
-You want to use those starting with `alma-` to add your own css style.
+You can see the list by inspecting the element with your dev tools. Don't use the classes that have an unpronounceable name, they will change at each new version of the widget.
```

Comment:
Is the `alma-` thing not relevant ?
It felt a bit more precise and objective than pronounceable/unpronounceable (even though I understand what you mean)